drink/src/lib.rs

```diff
@@ -50,11 +47,10 @@ pub type EventRecordOf<T> =
     EventRecord<<T as frame_system::Config>::RuntimeEvent, <T as frame_system::Config>::Hash>;
 
 /// A sandboxed runtime.
-pub struct Sandbox<R: Runtime> {
+pub struct Sandbox<R: frame_system::Config> {
```

Comment:
if we are abandoning runtime bounds, we should just be generic over account id for `mock_registry`

Reply:
with still need to be generic over some kind of runtime for the various implementation that will use the generic bound.

Ideally we move the mock stuff somewhere else since they are specific to contracts runtime